docs/USE-CASES.adoc

```diff
@@ -0,0 +1,35 @@
+= _Essentials_ Use Cases
+:toc:
+:sectnums:
+
+This document summarizes the Priority 1 use cases for _Jenkins Essentials_.
+
+NOTE: This is still early work.
+To help the thought process, the cases definition and analysis was done by thinking about what a demonstration would need to go through.
+
+== Priority 1 Cases: Risk analysis
+
+=== I can start an instance from scratch and it's available
+
+* Registration & authentication service need to be available:
+** An Internet connection is required
+** Bandwidth needs to be enough to download things in a _reasonable_ time.
+(It can probably be more than 5 minutes, if there is an issue.)
+* Binaries repository must be available.
+
```

Comment:
Missing something about how to deal with network errors: 
* Are the operations needed for first start and setup atomic? 
** If not, in case of network error, does the operations resume? Are there any sanity checks of the downloaded resources?
* Does the first start ad configuration have a default timeout?

Reply:
Wow, time flies. 

So, yeah @raul-arabaolaza as I tried to explain in the PR summary above, but I realize I was unclear, I was actually mainly here focusing on identifying the Use Case. I was planning to dig more into risk analysis in a followup PR.